src/Illuminate/Container/ContextualBindingBuilder.php

```diff
@@ -81,4 +81,20 @@ public function giveTagged($tag)
             return is_array($taggedServices) ? $taggedServices : iterator_to_array($taggedServices);
         });
     }
+
+    /**
+     * Define configuration key to be used to look up in configuration to bind as a primitive.
+     *
+     * @param string  $key
+     * @param ?string $default
```

Comment:
bad cs

Reply:
Bad CS because the first line of the docblock should wrapped?

Reply:
My bad on this. My guess might be there are two spaces between `string` and `$key` instead of just one? Or maybe the long line. :) Either way, my bad. I had actually fixed a few other things that the styleci bot complained about and figured it was all good since it passed.

Reply:
@simensen I found [this](https://laravel.com/docs/8.x/contributions#phpdoc) which says two spaces between @param and the type and then two spaces before the variable name so it could be that.